lua-resty-websocket-kong-0.1.0-1.rockspec

```diff
@@ -0,0 +1,24 @@
+package = "lua-resty-websocket-kong"
```

Comment:
We don't need the `-kong` suffix; the pgmoon fork is a good example to follow: https://github.com/Kong/pgmoon/blob/kong-dev/pgmoon-2.2.1-1.rockspec

Reply:
This is good to know! At a glance the EE repo today has custom pgmoon installation code littered throughout :( could very well be for a plethora of reasons other than naming, but I think I'm leaning towards keeping the suffix for now.

Reply:
Could keeping the same install path and different names lead to file overwriting if someone installs both packages?

Reply:
@locao Sounds possible but unlikely. Because lua-resty-websocket is bundled with OpenResty and not packaged by them (though it does look like somebody's made an [unofficial package](https://luarocks.org/modules/Invizory/lua-resty-websocket)), I don't think there should be many folks out there who are managing this library via luarocks currently.